### PR TITLE
Added notice

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
Screen Shot 2022-01-04 at 10 25 17 AM

### CODE REVIEW

File: apps/web/src/components/submission/components/Review.tsx [hunk 1]
```diff
@@ -154,6 +157,14 @@ export const Review: React.FC = () => {
           </Notice>
           <Error name='confirm' />
         </div>
+        <div className='bg-bcLightBlueBackground border-l-1 border-bcBluePrimary p-6 pb-5 md:pb-6 mb-5 text-left flex items-center'>
+          <div className='px-7 text-bcBluePrimary'>
+            <FontAwesomeIcon icon={faExclamationCircle}></FontAwesomeIcon>
+          </div>
+          <div className='text-bcBluePrimary'>
+            Please print and save this page for your own recrods.
```
Comment (arranfw) [inline apps/web/src/components/submission/components/Review.tsx:165]:
```suggestion
            Please print and save this page for your own records.
```

**Review by arranfw — CHANGES_REQUESTED** [review]
We should be able to match designs a bit closer with some changed margins, border radius and icon size. I think it could be worthwhile to abstract this into an alert component (with a single variant for now)

Placing the alert in the same div as the checkbox should help with spacing and will prevent the extra divider from showing

File: apps/web/src/components/Advisory.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const Advisory: React.FC = ({ children }) => {
```
Comment (arranfw) [inline apps/web/src/components/Advisory.tsx:4]: Can we call this `Alert` to match the BC gov term? https://developer.gov.bc.ca/Design-System/Alert-Banners

**Review by arranfw — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Advisory.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div className='bg-bcLightBlueBackground py-4  text-left flex items-center'>
```
Comment (arranfw) [inline apps/web/src/components/Advisory.tsx:6]: the `rounded` class should be added here

File: apps/web/src/components/submission/components/Review.tsx [hunk 4]
```diff
@@ -154,6 +156,7 @@ export const Review: React.FC = () => {
           </Notice>
           <Error name='confirm' />
         </div>
+        <Advisory>Please print and save this page for your own records.</Advisory>
```
Comment (arranfw) [inline apps/web/src/components/submission/components/Review.tsx:159]: Moving this up one line to be after the error will help the layout

File: apps/web/src/components/Advisory.tsx (hunk 3 above)
Comment (dbayly-freshworks) [reply apps/web/src/components/Advisory.tsx:6]: I thought about that, but the design doesn't have rounded corners.

Comment (arranfw) [reply apps/web/src/components/Advisory.tsx:6]: I see rounded on the version of designs I have 🤔  
image
https://www.figma.com/file/Cf5M3yWTLdJxngU6l6o17c/EHPR---Mockups?node-id=13%3A1211

File: apps/web/src/components/Advisory.tsx (hunk 2 above)
Comment (dbayly-freshworks) [reply apps/web/src/components/Advisory.tsx:4]: Sure, but I feel weird about it for a couple reasons. 
1. Alert is already an html tag name, and does something very different 
2. The BC gov alert component has a lot more features/functions that what this component does. It feels like either a mismatched component or a fair bit of scope creep.

File: apps/web/src/components/Advisory.tsx (hunk 3 above)
Comment (dbayly-freshworks) [reply apps/web/src/components/Advisory.tsx:6]: I see what happened, I had the component selected in figma, so it looked like it had short corners. It does feel a bit weird that this has rounded corners but the component above as sharp corners. 🤷 
 
Screen Shot 2022-01-04 at 10 20 47 AM

File: apps/web/src/components/Advisory.tsx (hunk 2 above)
Comment (arranfw) [reply apps/web/src/components/Advisory.tsx:4]: It's just different variants and accessibility features isn't it? I think we should implement as much as is reasonable, accessibility stuff is pretty important but we don't need the other variants for now (warning/error/etc..)

File: apps/web/src/components/submission/components/Review.tsx [hunk 5]
```diff
@@ -22,6 +23,7 @@ import {
   deploymentDurationOptions,
   placementOptions as allPlacementOptions,
 } from '../validation/preferences';
+import { Alert } from '../../Alert';
```
Comment (arranfw) [inline apps/web/src/components/submission/components/Review.tsx:26]: Last small comment, could you add alert to `components/index.ts` and import as `from '@components'`

**Review by arranfw — APPROVED** [review]
🥳

**Pull request merged**